### DISCUSSION TITLE
Significantly more stutters when the computer has been up for a long time

### DISCUSSION THREAD

**Discussion by refparo** [ppy/osu#23153] [Q&A]
I am used to keep my computer up for a long time, i.e. without shutting down. Instead, I make my computer sleep when I'm not using it. I noticed that osu! stutters significantly more frequently when the computer has been up for a long time, compared to when the computer is just booted. The system is Windows 11, and by "long time" I mean from several days to one or two weeks. Is there anyone who has similar experience?

**Comment by peppy** [comment]
This is probably nothing to do with osu!. It could be due to memory usage over time, or some process using more resources the longer it's open.

**Reply by refparo** [reply]
I have considered that possibility, but the performance drop isn't experienced in any other programs… Maybe it's because osu! is very fps-sensitive, idk…
